designer/client/src/PageCreate.tsx

```diff
@@ -4,6 +4,7 @@ import {
   hasComponents,
   hasNext,
   isQuestionPage,
+  type PageRepeat,
```

Comment:
Can't remember if ESLint does it, but mind grouping all the type imports together A-Z?